source/includes/steps-create-query-via-builder.yaml

```diff
@@ -45,6 +45,6 @@ title: Run the query
 level: 4
 ref: run-query
 content: |
-   To run the query, click the :guilabel:`Apply` button. You can clear
+   To run the query, click the :guilabel:`Analyze` button. You can clear
```

Comment:
We usually don't specify the UI element unless it is unclear which element you mean (i.e. two elements named "Find": one link and one button).